src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
```

Comment:
You can use `const` for `totalCost` since it's not being reassigned but only modified, which is allowed for `const` variables. Refer to checklist item 3.